docs/source/conf.py

```diff
@@ -191,3 +191,19 @@
 if current_version in SPHINX_DESIGN_VERSIONS:
     extensions.append("sphinx_design")
     html_js_files = ['fix-cards.js']
+
+linkcheck_ignore = [
+    # Ignore links that originate from the Kubernetes objects' documentation we have no control over.
+    'http://kubernetes.io/docs/user-guide/annotations',
+    'http://kubernetes.io/docs/user-guide/labels',
+    'https://examples.k8s.io/mysql-cinder-pd/README.md',
+    'https://examples.k8s.io/volumes/cephfs/README.md',
+    'https://examples.k8s.io/volumes/glusterfs/README.md',
+    'https://examples.k8s.io/volumes/iscsi/README.md',
+    'https://examples.k8s.io/volumes/rbd/README.md',
+    'https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources',
+    'https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds',
```

Comment:
why list all of those explicitly instead of using shared prefixes?

Reply:
IMO it's better to be explicit and ignore only specific links than ignore more than intended accidentally. 

Perhaps for `examples/k8s.io/volumes` we could use a prefix, as we can see `volumes` were completely removed. Still, I'd leave it as is due to the reason I mentioned (I can imagine `volumes` will be brought back, and we could ignore something we didn't intend).